superset-frontend/src/SqlLab/components/QueryTable/index.jsx

```diff
@@ -259,10 +273,12 @@ const QueryTable = props => {
             placement="bottom"
           >
             <span>
-              <StatusIcon
-                name={statusAttributes[q.state].config.name}
-                status={statusAttributes[q.state].config.status}
-              />
+              {q.state && (
```

Comment:
Is it possible for q.state to be undefined? You added this when we were testing, is this still needed?

Reply:
no, I changed it back